apps/server/src/services/TimerService.ts

```diff
@@ -24,9 +26,10 @@ export class TimerService {
    * @param {object} [timerConfig]
    * @param {number} [timerConfig.refresh]
    */
-  constructor(timerConfig?) {
+  constructor(timerConfig?: { refresh?: number; updateInterval?: number }) {
     this._clear();
-    this._interval = setInterval(() => this.update(), timerConfig?.refresh || 1000);
+    this._interval = setInterval(() => this.update(), timerConfig?.refresh ?? 1000);
+    this._updateInterval = timerConfig?.updateInterval ?? 1000;
```

Comment:
Nitpicking, but you can give `timerConfig` a default value of `{}` and not have to deal with the (subjectively) ugly `?` operator:

```ts
constructor(timerConfig?: { refresh?: number; updateInterval?: number } = {}) {
    this._clear();
    this._interval = setInterval(() => this.update(), timerConfig.refresh ?? 1000);
    this._updateInterval = timerConfig.updateInterval ?? 1000;
}
```